### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by kart-c**
## Video Listing

- Add videos to the database
- Fetch videos on initial render
- Line clamp 2 for video title
- Add timestamp on video card

### CODE REVIEW

File: src/assets/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import igcompany from './igcompany.jpg';
+import PlayStation from './PlayStation.jpg';
+import Throneful from './Throneful.jpg';
+import Shirrako from './Shirrako.jpg';
+import Fennec from './Fennec.jpg';
+import DIZEUUL from './DIZEUUL.jpg';
+import MrEdxwx from './MrEdxwx.jpg';
+import xbox from './xbox.jpg';
+import ign from './ign.jpg';
+import rockstar from './rockstar.jpg';
```
Comment (AtulPant2704) [inline src/assets/index.js:16]: Try to have same naming convention of all of your images.

**Review by AtulPant2704 — APPROVED** [review]
The UI looks really cool, good job.

**Review by AnkurChunekar — APPROVED** [review]

**Comment by AnkurChunekar** [comment]
UI looks good code is neat and clean

File: src/assets/index.js (hunk 1 above)
Comment (kart-c) [reply src/assets/index.js:16]: I have used the video channel names to avoid confusion in the database while using these values.

**Pull request merged**